src/Compilers/Core/Portable/Operations/IMultiVariableDeclaration.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents multiple declared variables in a single declarator.
+    /// </summary>
+    /// <para>
+    /// Current Usage:
+    ///   (1) VB As New statements
+    ///   (2) VB multiple declarations in a single declarator
+    /// </para>
```

Comment:
></para> [](start = 8, length = 7)

Consider adding an explanation that the initializer applies to all variables.

Reply:
Added a remark about initializer presence.

---
In reply to: [146966663](https://github.com/dotnet/roslyn/pull/22842#discussion_r146966663) [](ancestors = 146966663)